.gitignore

```diff
@@ -85,8 +85,12 @@ cache/
 exported_videos/*
 /randovania/data/icons/executable_icon.ico
 
+# Ignore generated UI files
+/randovania/gui/ui_files/*_ui.py
+
 # Ignore certain files on root
 /*.rdvgame
 /*.rdvpreset
+/*.log
 /*.txt
-/*.log
\ No newline at end of file
+!requirements*.txt
```

Comment:
I would rather know why .txts are in the gitignore in the first place.

Reply:
Common mistake of adding files to root.
Since we don't plan on addingang other txt files to root we can just ignore.